dialogue/Tutorial/TutJump.json

```diff
@@ -13,7 +13,7 @@
     ],
     "pl": [
       "Tym szybczej biegasz, tym wyżej możesz skakać.",
-      "Przetrzymanie przecisk skoku jeszcze maksymalizuje Twóją wysokość skoku.",
+      "Przetrzymanie przecisk skoku jeszcze maksymalizuje Twoją wysokość skoku.",
       "Stukanie przycisk skoku raczej spowoduje mniejsze skoki!"
```

Comment:
"Szybkie kliknięcie przycisku skoku da raczej mniejszy efekt!" Changed the sentence a little bit, but still means the same thing and I think it sounds better